### PR TITLE
Editorial: use "exception name" DOMException notation

### PR DESCRIPTION
**Pull request by tobie**
This notably helps avoid confusion with ES errors (e.g. SyntaxError).

***
[Preview](https://s3.amazonaws.com/pr-preview/tobie/dom/domexceptions.html) | [Diff](https://s3.amazonaws.com/pr-preview/whatwg/dom/9ada239...tobie:ab54d5d.html)

### CODE REVIEW

File: dom.bs [hunk 1]
```diff
@@ -261,7 +261,8 @@ against a <var>set</var>, run these steps:
  <li><p>Let <var>s</var> be the result of <a>parse a selector</a> <var>selectors</var>.
  [[!SELECTORS4]]
 
- <li><p>If <var>s</var> is failure, then <a>throw</a> a {{SyntaxError}}.
+ <li><p>If <var>s</var> is failure, then <a>throw</a> a "{{InvalidCharacterError!!exception}}"
```
Comment (annevk) [inline dom.bs]: This should be SyntaxError.

File: dom.bs [hunk 2]
```diff
@@ -294,18 +295,19 @@ run these steps:
...
  <li>If either <var>qualifiedName</var> or <var>prefix</var> is
  "<code>xmlns</code>" and <var>namespace</var> is not the
  <a>XMLNS namespace</a>, then <a>throw</a> a
- {{NamespaceError}}.
+ "{{NamespaceError!!exception}}" {{DOMException}}.
 
  <li>If <var>namespace</var> is the <a>XMLNS namespace</a> and neither <var>qualifiedName</var>
- nor <var>prefix</var> is "<code>xmlns</code>", then <a>throw</a> a {{NamespaceError}}.
+ nor <var>prefix</var> is "<code>xmlns</code>", then <a>throw</a> a
+ "{{InvalidStateError!!exception}}" {{DOMException}}.
```
Comment (annevk) [inline dom.bs]: This should be NamespaceError.

File: dom.bs [hunk 3]
```diff
@@ -2054,37 +2056,37 @@ of a <var>node</var> into a <var>parent</var> before a <var>child</var>, run the
...
  <a>node</a>,
  <a>throw</a> a
- {{HierarchyRequestError}}.
+ "{{HierarchyRequestError!!exception}}" {{DOMException}}.
 
  <li>If <var>node</var> is a
  <a>host-including inclusive ancestor</a>
  of <var>parent</var>, <a>throw</a> a
- {{HierarchyRequestError}}.
+ "{{NamespaceError!!exception}}" {{DOMException}}.
```
Comment (annevk) [inline dom.bs]: HierarchyRequestError

File: dom.bs [hunk 4]
```diff
@@ -5328,11 +5332,14 @@ invoked, must run these steps:
     <tr><td>"<code>uievents</code>"
   </table>
 
- <li><p>If <var>constructor</var> is null, then <a>throw</a> a {{NotSupportedError}}.
+ <li>
+  <p>If <var>constructor</var> is null, then <a>throw</a> a "{{NotSupportedError!!exception}}"
+  {{DOMException}}.
```
Comment (annevk) [inline dom.bs]: There's no need for a newline before the `<p>` here.

File: dom.bs (hunk 1 above)
Comment (tobie) [reply dom.bs]: Very sorry.

**Pull request merged**